ng-dev/pr/common/fetch-pull-request.ts

```diff
@@ -193,6 +222,32 @@ export function getStatusesForPullRequest(
   };
 }
 
+/**
+ * Checks for `TESTED=[reason]` review comment on a current commit sha from a google organization member
+ */
+export async function pullRequestHasValidTestedComment(
```

Comment:
Can we move this function into assert-enforce-tested.ts?

Reply:
Yup. Will do.